File: uri/soma/soma.scala
object Main{
    def soma( a:Int , b:Int) : Int = 
        a + b
    
    def main(args: Array[String]){
        var a = scala.io.StdIn.readInt() : Int
        var b = scala.io.StdIn.readInt() : Int
        var res = soma(a, b)
        printf("SOMA = %d\n", res)
    }
}

File: uri/soma/soma.js
/*
var input = require('fs').readFileSync('/dev/stdin','utf8');
var lines = input.split('\n');

function soma(a, b){
    return(a+b);
}

var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
console.log("SOMA ="+ soma(a,b));

*/

(function(){

    function soma(a, b){
        return(a+b);
    }

    var a = 15;
    var b = 30;
    console.log("SOMA = "+ soma(a,b));
}).call();